packages/carbon-component-mapper/src/files/date-picker.d.ts

```diff
@@ -0,0 +1,13 @@
+import { FormGroupProps } from "./form-group";
+import { UseFieldApiComponentConfig, AnyObject } from "@data-driven-forms/react-form-renderer";
+
+interface InternalDatePickerProps extends AnyObject, React.HTMLProps<HTMLInputElement> {
```

Comment:
This does not extend carbon date picker props.

Reply:
@Hyperkid123 Carbon is not using typescript to define props.

Reply:
:facepalm: